modules/tensor_mechanics/include/materials/ComputeCrackedStress.h

```diff
@@ -0,0 +1,90 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#ifndef COMPUTECRACKEDSTRESS_H
+#define COMPUTECRACKEDSTRESS_H
+
+#include "Material.h"
+#include "RankTwoTensor.h"
+#include "RankFourTensor.h"
+#include "RotationTensor.h"
+#include "DerivativeMaterialInterface.h"
+
+class ComputeCrackedStress;
+
+template <>
+InputParameters validParams<ComputeCrackedStress>();
+
+/**
+ * Phase-field fracture
+ * This class computes the stress and energy contribution to fracture
+ * Small strain Anisotropic Elastic formulation
+ * Stiffness matrix scaled for heterogeneous elasticity property
+ */
+class ComputeCrackedStress : public DerivativeMaterialInterface<Material>
+{
+public:
+  ComputeCrackedStress(const InputParameters & parameters);
+
+protected:
+  virtual void computeQpProperties();
+  virtual void initQpStatefulProperties();
+
+  /// Base name of the stress and strain modified to include cracks
+  const std::string _uncracked_base_name;
+
+  /// Use current value of history variable
+  bool _use_current_hist;
+
+  const MaterialProperty<RankTwoTensor> & _mechanical_strain;
+
+  /// Uncracked stress calculated by another material and modified here to include cracks
+  const MaterialProperty<RankTwoTensor> & _uncracked_stress;
+
+  /// Uncracked Jacobian_mult calculated by another material and modified Here
+  const MaterialProperty<RankFourTensor> & _uncracked_Jacobian_mult;
```

Comment:
Did you modify the _uncracked_Jacobian_mult? If not, we should delete 'modified here' from the comment.